control-station/src/services/PodSocketClient.ts

```diff
@@ -16,6 +16,7 @@ interface ServerToClientEvents {
 	connect: () => void;
 	disconnect: (reason: Socket.DisconnectReason) => void;
 	serverResponse: (data: string) => void;
+	faulted: (data: string) => void;
```

Comment:
Chore: since we named the emit message `fault`, we'll need to use the same message name here
```suggestion
	fault: (data: string) => void;
```